content/en/docs/setup/_index.md

```diff
@@ -100,7 +100,7 @@ The following production environment solutions table lists the providers and the
 | [Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm)          | &#x2714;             |   &#x2714;     |             |      |          |
 | [oVirt](https://www.ovirt.org/)         |              |        |             |     | &#x2714;          |
 | [Pivotal](https://pivotal.io/) | | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service) | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service)           |      |          |
-| [Platform9](https://platform9.com/)         | &#x2714;             | &#x2714;      | &#x2714;            |      |   &#x2714;       |&#x2714;
+| [Platform9](https://platform9.com/)         | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)             | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)      | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)            |  [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)    |   [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)       |[Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)
```

Comment:
Hi @avigailof, having the Platform9 Managed Kubernetes offering listed in both the "Managed Kubernetes" and "Turnkey Cloud" as well as "On-prem datacenter" seems like a bit of a stretch. Either it's managed or it isn't, no?

Reply:
Hi @jaypipes, I can see where you're coming from, but Platform9 does offer a managed solution that can be deployed on-premises, in the public cloud or custom/edge environments. Similar to VMware PKS, Docker or Rancher 2.0 listed as "turn-key" -- we support a similar deployment target, we just happen to have the same product name for all these use cases. Our offering emphasizes enabling hybrid, mixed environments under centralized management. It is fully-managed remotely with our cloud management plane, on any environment. Hope this clarifies it. please let me know if I can help answer any questions or if you want to discuss further.

Reply:
> It is fully-managed remotely with our cloud management plane, on any environment. Hope this clarifies it. please let me know if I can help answer any questions or if you want to discuss further.

I would list it in Managed and On-prem Datacenter, then.

Reply:
> @jaypipes thanks for the comments, perhaps we don't understand the definition of **_custom_**, here is an explanation of what we do
> ​
> W.r.t Custom (Cloud, Baremetal, VMs: Platform9 Managed Kubernetes supports all of these infrastructures)
> ​
> 
>     * Control plane is deployed into the customer's infrastructure (their Cloud environment, Virtual Machines or on Bare metal servers)
> 
>     * Extensive configuration is allowed for the control plane (e.g changing authentication provider, changing the number of pods etc.)
> 
>     * Customers are able to out of band management like backup or monitoring (in addition to what Platform9 Managed Kubernetes provides)
> 
>     * Platform9 Managed Kubernetes provides the necessary monitoring like health checks, performance monitoring
>       ​
>       As you can see we do allow customer to change the control plane and hence looking to add ourselves to the **_Custom_** category. Let us know if we misunderstood that.

Platform9 already has a check box in all categories except for "Custom (cloud)", though. What this PR does is converts all those checkmarks into a link to the Platform9 Managed Kubernetes landing page. As such, it just kind of feels a bit over-the-top to me.

Why not just add a checkmark to the "Custom (cloud)" category if it was important to be recognized in that particular category?